src/SDKs/Gsm/Management.Gsm/Microsoft.Azure.Management.Gsm.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.Management.Gsm</PackageId>
+		<Description>Microsoft.Azure.Management.Gsm Library</Description>
+		<AssemblyName>Microsoft.Azure.Management.Gsm</AssemblyName>
+		<Version>1.0.0-preview</Version>
+    <PackageTags>Gsm;</PackageTags>
+		<PackageReleaseNotes>
+			<![CDATA[
+        This is a public preview release of the Azure Gsm SDK. Included with this release is all operation to get/modify partner id for tenant
+      ]]>
+		</PackageReleaseNotes>
+	</PropertyGroup>
+	<PropertyGroup>
+		<TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+	</PropertyGroup>
+	<ItemGroup>
+	  <Folder Include="Properties\" />
```

Comment:
Please remove, no need to add this directory explicitly

Reply:
@shahabhijeet  GSM stands for Global subscription management, it is also our web role name.
Thanks for reviewing the code, fix the comments. 
This is our spec repo pr: https://github.com/Azure/azure-rest-api-specs/pull/2285